channeldb/control_tower_test.go

```diff
@@ -363,13 +364,31 @@ func assertPaymentStatus(t *testing.T, db *DB,
 
 	t.Helper()
 
-	pStatus, err := db.FetchPaymentStatus(hash)
+	var paymentStatus = StatusGrounded
+	err := db.View(func(tx *bbolt.Tx) error {
```

Comment:
Why doesn't this use `fetchPaymentStatus`?

Reply:
It does now 😉 https://github.com/lightningnetwork/lnd/pull/2761/commits/6dfa706101efad799b468385a306971dfe326040